wagtail/models/__init__.py

```diff
@@ -2305,6 +2306,34 @@ def get_cached_paths(self):
         """
         return ["/"]
 
+    def get_cache_key_components(self):
+        """
+        The components of a ``Page`` which make up the ``cache_key``. Any change to a
+        page should be reflected in a change to at least one of these components.
+        """
+
+        return [
+            self.id,
+            self.url_path,
+            self.last_published_at.isoformat() if self.last_published_at else None,
```

Comment:
Just want to flag that it is possible to programatically update a live page without increasing this (and it could trip up people trying to make urgent updates) - is it worth noting that in the docs, or adding an explicit cache purge somewhere?

Reply:
Explicit cache purging is difficult, as it'd get difficult to know whether you're clearing the cache before or after the update, or even what the final cache key ended up being. I agree that we should document it.